src/components/InviteVideo.tsx

```diff
@@ -0,0 +1,23 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import { getPrivateVideoInviteRef } from "../connectors";
+import { storageRef } from "../firebaseInit";
+import Loading from "./Loading";
+import Video from "./Video";
+
+interface Props {
+  readonly userId: string;
+  readonly memberId: string;
+}
+
+export default class InviteVideo extends React.Component<Props, {}> {
+  public render() {
+    // TODO group by uid instead of mid?
+    const videoUrl = `https://storage.googleapis.com/raha-video/${this.props.memberId}/invite.mp4`
```

Comment:
@rahulgi after update this will change to props.userId ?

Reply:
currently sticking with mid